FleekHQ/s3x
# Fix status code missing

## Description
fleek-S3x was logging errors which we dont want to log for our audit logs

## Motivation and Context
Minio seems to have had some breaking change a while ago where their audit logs are not logging the status code from the http request, that makes it so we are unable to determine if the request was a success or an error.

https://app.clubhouse.io/terminalsystems/story/15217/hotfix-for-s3x-logging-errors

## How to test this PR?
Create a bucket that doesnt exist. Log Payload should show up on terminal
Create the same bucket again, log payload should not be logged.
